modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -4448,6 +4506,9 @@ private void _validateWorkflowAction(
 		new CentralizedThreadLocal<>(
 			ObjectEntryLocalServiceImpl.class + "._skipModelListeners",
 			() -> false);
+	private static final Snapshot<TreeFactory> _treeFactorySnapshot =
+		new Snapshot<>(
+			ObjectEntryLocalServiceImpl.class, TreeFactory.class, null, true);
```

Comment:
Because in [TreefactoryImpl I call this one](https://github.com/liferay-objects/liferay-portal/blob/18582429d311a0e6232a08178a7670c0a67af021/modules/apps/object/object-service/src/main/java/com/liferay/object/internal/tree/TreeFactoryImpl.java#L109-L110)